sites/fast-website/src/app/components/side-navigation/side-navigation.styles.ts

```diff
@@ -2,56 +2,54 @@ import { css } from "@microsoft/fast-element";
 import { display } from "@microsoft/fast-foundation";
 import {
     accentForegroundActiveBehavior,
+    accentForegroundCutRestBehavior,
     accentForegroundHoverBehavior,
-    neutralForegroundRestBehavior,
 } from "@microsoft/fast-components";
 
 export const SideNavigationStyles = css`
-    ${display("flex")}:host {
-        --engaged-height: calc(var(--design-unit) * 9px);
+    ${display("flex")} :host {
+        --active-height: calc(var(--design-unit) * 13px);
```

Comment:
The references to --design-unit here seem somewhat arbitrary in terms of what we are multiplying by. Do we want this to change size in coordination with the design unit, or should we just hardcode the value?

Reply:
@chrisdholt My thinking was that if I used design unit as part of the height that then the scroll indicators would adjust with the rest of the design system as we adjusted the page's density. If this is not the case I will hard code them.

Reply:
@SamanthaAO I'm primarily wondering if we've considered or seen how vast that difference can be. Could this end up obstructing the UI, etc. Technically, I'm not sure changing the density multiplier would directly affect this. Changing the design unit certainly would. A small change to the design unit could introduce significant changes here. I think we could certainly look at making these adaptive, I'm just not sure that we have enough insight into how we should go about that now.

Reply:
That makes sense. I think that the safest decision now would be to hard code these values. I will update it.